components/section-separator/index.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+const Separator = styled.hr`
+  width: calc(100% - 40px);
+  margin: 6rem 0;
+  border-color: #000;
+`
+
+const SectionSeparator = () => <Separator />
```

Comment:
just small issue here for this component, there is no need to create a separate functional compoent after using styled.hr. you can simply rename `Separator` to `SectionSeparator` and remove the functional component wrapper.

The reason is that `SectionSeparator` as it is right now cannot be extended with extra styles by its users because it doesn't pass props down.

In case you needed it in the future, here's an extensible implementation of the component, but in this case I suggest just exporting the output of `styled.hr`.

```
const SectionSeparator = ({ myProp, ...otherProps }) => {
  // custom logic with myProp
  return <Separator {...otherProps} />
}
```

Reply:
Right. Good point, I have not worked with this kind of framework yet but I think I am starting to understand how it works.
Thanks for the comment.